langserve/server.py

```diff
@@ -681,10 +681,23 @@ async def _stream() -> AsyncIterator[dict]:
                     ) from validation_exception
 
             try:
+                config_w_callbacks = config.copy()
+                event_aggregator = AsyncEventAggregatorCallback()
+                config_w_callbacks["callbacks"] = [event_aggregator]
```

Comment:
ooc: should we be merging rather than overwriting these?

Reply:
I'll defer to @eyurtsev since I'm copying his pattern here

Reply:
Historically was fine to overwrite. With addition of per request modifier, we might want to merge if a user is adding custom callbacks

Reply:
I'd say keep as is for now and we can follow up with a separate PR to merge

Reply:
works for me, will merge